ext/gmp/gmp.c

```diff
@@ -1017,11 +1033,11 @@ ZEND_FUNCTION(gmp_strval)
 	/* Although the maximum base in general in GMP is 62, mpz_get_str()
 	 * is explicitly limited to -36 when dealing with negative bases. */
 	if ((base < 2 && base > -2) || base > GMP_MAX_BASE || base < -36) {
-		php_error_docref(NULL, E_WARNING, "Bad base for conversion: " ZEND_LONG_FMT " (should be between 2 and %d or -2 and -36)", base, GMP_MAX_BASE);
-		RETURN_FALSE;
+		zend_argument_value_error(2, "must be between 2 and %d or -2 and -36", GMP_MAX_BASE);
```

Comment:
```suggestion
		zend_argument_value_error(2, "must be between 2 and %d, or -2 and -36", GMP_MAX_BASE);
```
does the `,` work here?